amazon-cloudwatch-container-insights/k8s-yaml-templates/cwagent-kubernetes-monitoring/cwagent-daemonset.yaml

```diff
@@ -15,7 +15,8 @@ spec:
     spec:
       containers:
         - name: cloudwatch-agent
-          image: amazon/cloudwatch-agent:1.230621.0
+          image: public.ecr.aws/a3i2u7g3/cw-agent-seth:upgrade-telegraf-v1-21-4
```

Comment:
same sentiment as other comments. I understand that this is an image that works, but maybe what we need is to point to the `latest` image in ECR